charts/invenio/templates/_helpers.tpl

```diff
@@ -283,7 +301,7 @@ Get the sentry secret name
 {{- else if  .Values.invenio.sentry.secret_name -}}
   {{- print .Values.invenio.sentry.secret_name -}}  
 {{- else -}}
-  {{- "sentry-secrets" -}}
+  {{- printf "%s-%s" (include "invenio.fullname" .) "sentry-secrets" -}}
```

Comment:
A comment I think holds true for multiple places:

Your change looks good. But I dislike the old name `sentry-secrets` that we keep. I think having the resource kind in the resource name is redundant; in my experience it's always clear from context or metadata if it's a secret or some other object. I think it looks nicer to have a Secret called "invenio-sentry" than a secret called "invenio-sentry-secrets".

What do you think?

Reply:
I think the sentry secret and the ingress are the only resources that include the resource type in the name actually.
The secret that contains the secret keys and salts is called `invenio.fullname-secrets`, but I am inclined to think this is correct, I don't know what else we could call it.

Reply:
Yeah, it's very hard to know whether to make yet another improvement or keep the change simpler. I'm not sure either, but if this occurred to both of us then that's probably a sign that it might be worthwhile.

Reply:
> The secret that contains the secret keys and salts is called `invenio.fullname-secrets`, but I am inclined to think this is correct, I don't know what else we could call it.

I'm inclined to say that it should simply be called `invenio.fullname`. Same logic as before; I already know it's a Secret. But it's fine by me if you want to keep the `-secrets` suffix, there's clearly no harm to it.

Reply:
I think everything else here looks great, so just tell me if you want to keep the suffix or not and then I'll approve.

Reply:
I changed the secret name to just `invenio.fullname`, you convince me 😅